README.md

```diff
@@ -792,21 +782,50 @@ See [openhab-js : actions.Voice](https://openhab.github.io/openhab-js/actions.ht
 
 #### Cloud Notification Actions
 
-Note: Optional action if [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/) is installed.
+Requires the [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/) to be installed.
 
 Notification actions may be placed in rules to send alerts to mobile devices registered with an [openHAB Cloud instance](https://github.com/openhab/openhab-cloud) such as [myopenHAB.org](https://myopenhab.org/).
 
-For available actions have a look at the [Cloud Notification Actions Docs](https://www.openhab.org/docs/configuration/actions.html#cloud-notification-actions).
+There are three different types of notifications:
+
+- Broadcast Notifications: Sent to all registered devices and shown as notification on these devices.
+- Log Notifications: Only shown in the notification log, e.g. inside the Android and iOS Apps.
+- Standard Notifications: Sent to the registered devices of the specified user and shown as notification on his devices.
+
+To send these three types of notifications, two notification builders are available:
+
+- `.logNotificationBuilder(message)`: Creates a builder to send a log notification with the given message.
+  - `.withIcon(icon)`: Sets the icon of the notification.
+  - `.withSeverity(link)`: Sets the severity of the notification.
+  - .`send()`: Sends the notification.
+- `.notificationBuilder(message)`: Creates a builder to send a broadcast or standard notification with the given message.
+  - all methods of the log notification builder plus:
+  - `.withUserId(userId)`: By specifying the mail address of a specific openHAB Cloud user, the notification is sent only to this user.
+  - `.withOnClickAction(onClickAction)`
+  - `.withMediaAttachmentURL(mediaAttachmentURL)`: Sets the URL of a media attachment to be displayed with the notification. This URL must be reachable by the push notification client.
+  - `.withActionButton1(actionButton1)`
+  - `.withActionButton2(actionButton2)`
+  - `.withActionButton3(actionButton3)`
```

Comment:
I'd prefer `addActionButton()`, so the same function can be called up to three times. Maybe raise an exception when called more than three times?

Reply:
Sounds good.